### DISCUSSION TITLE
Help needed for my research

### DISCUSSION THREAD

**Discussion by elifuslums** [community/community#136845] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Dear All,

I am doing a master's at Nova IMS university, Portugal and my thesis is focused on the user behavior on Low Code Development Platforms and I would really appreciate if you could take a few minutes to fill out my thesis survey below:

https://novaims.eu.qualtrics.com/jfe/form/SV_6EfXHBB3K071mHY

Thanks in advance for your help!

**Comment by Yackens** [comment]
Hi, I have filled it in! Hope everything goes well.

**Reply by elifuslums** [reply]
Thank you!

**Reply by ebndev** [reply]
Hey all, thanks for being a part of the GitHub Community!

I'm going to unmark this as the answer so that others may be more likely to still click into the discussion and review your ask.

**Reply by elifuslums** [reply]
Thank you!

**Comment by Ammar-Alnagar** [comment]
Will do , good luck with your thesis!

**Reply by elifuslums** [reply]
Thank you so much!
